mods/lord/Blocks/lord_gems/src/gems.lua

```diff
@@ -0,0 +1,30 @@
+--lord_gems
+local S = minetest.get_mod_translator()
+
+minetest.register_craftitem("lord_gems:blue", {
+	description = S("Blue Gem"),
+	inventory_image = "lord_gems_blue.png",
```

Comment:
выравнивание
```suggestion
	description     = S('Blue Gem'),
	inventory_image = 'lord_gems_blue.png',
```
и ниже